docs/design/integrate-with-cc-tools/README.md

```diff
@@ -0,0 +1,308 @@
+
+###### Build the chaincode
+
+After setting up the development environment, you can build the chaincode using the [Ego](https://pkg.go.dev/github.com/edgelesssys/ego) framework which is necessary to run in encrypted enclaves. You can use the FPC build Makefile [here](https://github.com/osamamagdy/fabric-private-chaincode/blob/feat/create-sample-app/ecc_go/build.mk) but need to point the `CC_NAME` to it. The chaicode will be present as a docker image as the fabric network is using docker-compose.
+
+###### Start the Fabric network
+
+Now that we have the chaincode built and ready to be deployed, we need to start the fabric network before going any further. We're following the FPC [guide](https://github.com/osamamagdy/fabric-private-chaincode/tree/feat/create-sample-app/samples/deployment/test-network#prepare-the-test-network) for preparing and starting the network.
+
+###### Install the chaincode
+
+FPC provides another `installFPC.sh` script for installing the chaincode on the peers but first, we must make sure to set the `CC_ID` with the chaincode name and `CC_VER` with the path to the `mrenclave` of that chaincode. Then we run the script and also start the chaincode containers like [here](https://github.com/osamamagdy/fabric-private-chaincode/tree/feat/create-sample-app/samples/deployment/test-network#install-and-run-the-fpc-chaincode).
+
+###### Work with the client application
+
+As explained in the [client](#on-the-client-side-level) section we're following the standard by FPC so using a client application based on the FPC client SDK is the way to go. In [here](https://github.com/osamamagdy/fabric-private-chaincode/blob/feat/create-sample-app/samples/chaincode/simple-asset-go/README.md#invoke-simple-asset) is a guide on how to build and use an FPC client cli to communicate with the chaincode. It builds the tool, updates the connection configurations, initializes the enclave, and invokes the transactions. All needed is to edit the chaincode parameters like `CC_NAME`, `CHANNEL_NAME` , etc...
+
+## Limitations
+
+There are some standard fabric features that are not compatible with the security properties of FPC and how FPC works and then it's not part of our intention to support.
+
+* `GetTransient()` and `GetBinding()`. This is not needed as the application-level confidentiality is handled by the FPC client invocation approach
+* GetSignedProposal().
```

Comment:
I thought it is supported: https://github.com/hyperledger/fabric-private-chaincode/blob/main/ecc_go/chaincode/enclave_go/shim.go#L248

Reply:
Right. I missed it